### ISSUE TITLE
Image borders are ignored

### ISSUE THREAD

**Issue by bast** [coderefinery/github-without-command-line#31]
Probably we should use Markdown/Sphinx syntax anyway instead of HTML.

**Comment by rkdarst** [comment]
The raw <img> are being transformed to a directive via a regex, and it
doesn't processes those border properties at all (nor the widths).

I searched and didn't find a built-in way to add a border to the
images.  So, I ignored it for the time being.  I guess it should be
done in CSS.  I just found this:
  https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Images.html
which says `:class: with-border`, but that doesn't add a border, it
puts a CSS class (not sure if with-border is in the default css or
not), but it seems like closer to the right way to do things.  (maybe
in our case, it's better to have a border by default, and remove the
border only when desired?)

Adding the CSS is easy,
sphinx-lesson:sphinx_lesson/_static/sphinx_lesson.css

There *is* support for explicit widths of the figures, but I left that
off when setting things up.  Should we use that by default?  I just
had the idea that it's better to leave it off, and try to keep the
images narrow enough so that they aren't compressed too much when the
page is narrow - since people will be working on narrow screens
anyway.  I think the default is 1:1 pixel ratio.
